indictrans/rom2ind.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python 
+# -*- coding: utf-8 -*-
+
+#Copyright (C) 2015 Irshad Ahmad Bhat
+
+"""
+Transliteration Tool:
+Roman to Indic transliterator
+"""
+
+import re
+import sys
+import string
+import os.path
+import warnings
+
+import numpy as np
+
+from ._utils import wxilp
+from ._decode import viterbi
+
+warnings.filterwarnings("ignore")
+
+class rom_to_ind():
+    """Transliterates words from Roman to Indic script"""
+
+    def __init__(self, lang): 
+        self.lookup = dict()
+        self.tab = '\x01\x04'
```

Comment:
This is just to keep tabs as tabs and spaces as spaces after transliteration. I'm just mapping tabs and spaces to some highly infrequent strings ('\x01\x04' and '\x02\x04'') and later (after transliteration) convert these strings back to tabs and spaces. Note that I need to split the input sentence on spaces (both tabs and single spaces) so that I can transliteration the individual words and join them back by spaces after transliteration. If I use the simple `str.split()` command of python, I loose the evidence of tabs if there were any.